README.md

```diff
@@ -43,7 +43,7 @@ If you have any question, please use the [discussions page](https://github.com/n
 > If you want to use NixVim with nixpkgs 23.05 you should use the `nixos-23.05` branch.
 
 ### Without flakes
-NixVim now ships with `flake-compat`, which makes it usable from any system.
```

Comment:
I don't think that changed? I think flake-compat is the `default.nix` at the root of the projet

Reply:
Indeed. Roll back done.